php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -28,11 +28,41 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     public $form;
 
+    /**
+     * Variable defining the type of form in use
+     *
+     * Note: this seems to always be defined as 'XIN'
```

Comment:
@driusan should we set this to 'XIN' in here and avoid that every project sets it in every instrument ?

Reply:
That seems to be an assumption.. I think it's better to be explicit. Though to be honest, I'm not sure what this is used for. The only place I see it used in the code is:

```
        //XIN specific functionality
        if ($this->formType=="XIN") {
            $this->_nullStatus($values);
        }
```